crates/sui-archival/src/lib.rs

```diff
@@ -3,10 +3,6 @@
 #![allow(dead_code)]
 
 pub mod reader;
-pub mod writer;
-
-#[cfg(test)]
-mod tests;
```

Comment:
So the writer is being removed immediately? How does this impact the infra that we're running today for populating the archival? Or is this the archival that used to run inside the node and the one that runs as a part of ingestion framework still functions?

Reply:
yeah, that's the old writer. Data ingestion still works after the change